rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -548,6 +580,14 @@
     }
 
     $(function($) { // onLoad
+      // there can be only one
+      if (document.querySelector(tooltipSel) === null) {
+        var newTooltip = $(
+          '<div class="rapid-response-tooltip hidden"></div>'
+        );
```

Comment:
Alternate approach:

- Include this `<div>` in the actual underscore template and accept a template variable that indicates whether or not it should be hidden
- Add an empty container element in the HTML (or dynamically create one and prepend it to the body), and assign it to a top-level var
- On page load, render the template w/ the template var indicating that it should be hidden (`$tooltipContainer.html(tooltipTemplate({...}))`)